### ISSUE TITLE
meta-pipeline missing PULL_NUMBER env

### ISSUE THREAD

**Issue by cpwc** [jenkins-x/jx#5043]
### Summary 
The latest pipelinerunner comes default with `--use-meta-pipeline` enabled. However, `PULL_NUMBER` does not exist causing build to fail for me while trying to generate URL for preview.

### Jx version

The output of `jx version` is:

```
jx                 2.0.561
jenkins x platform 2.0.942
Kubernetes cluster v1.12.9
kubectl            v1.15.0
helm client        Client: v2.14.1+g5270352
git                2.19.1
Operating System   Mac OS X 10.14 build 18A391
```

### Jenkins type

- [x] Serverless Jenkins X Pipelines (Tekton + Prow)
- [ ] Classic Jenkins

### Kubernetes cluster

Kubernetes + KOPS

**Comment by hferentschik** [comment]
Hi @cpwc, thanks for raising the issue. Any chance you could try the latest version of `jx`? I think your issue might be related to issue #5064 which is fixed by now. It would be awesome if you could verify whether your problem still persist.

If it does, could you provide a bit more context? What is the error message in the failed build? What are the steps to reproduce it?

**Comment by cpwc** [comment]
@hferentschik most definitely. Let me do this and update this issue accordingly.

**Comment by hferentschik** [comment]
> most definitely. Let me do this and update this issue accordingly.

thanks

**Comment by cpwc** [comment]
@hferentschik 

upgraded my cluster to the latest version of jx and manually updated `pipelinerunner` from `prow` to pick up the latest build image. All `envs` are available to meta-pipeline. Thanks for fixing this.

I'll go ahead and close this issue. Duplicate of #5064. Resolved in 3a121b2.

/close

**Comment by hferentschik** [comment]
> upgraded my cluster to the latest version of jx and manually updated pipelinerunner from prow to pick up the latest build image. All envs are available to meta-pipeline. Thanks for fixing this.

Awesome, thanks for testing it out.
